tutorials/commands/ssh-across-clusters.rst

```diff
@@ -0,0 +1,138 @@
+.. _ssh-across-clusters:
+
+===================
+SSH across clusters
+===================
+
+Let's say you want to create a Flux instance in an allocation on a cluster (e.g., let's say out first cluster is "noodle") 🍜️
+and then connect to it via ssh from another cluster (let's say our second cluster is called "quartz"). This is possible with the right
+setup of your ``~/.ssh/config``. 
+
+----------------------
+Create a Flux Instance
+----------------------
+
+First, let's create the allocation on the first cluster. We typically want to ask for an allocation,
+and run flux start via our job manager. Here we might be on a login node:
+
+.. code-block:: sh
+
+    # slurm specific
+    $ salloc -N4 --exclusive
+    $ srun -N4 -n4 --pty --mpibind=off flux start
+
+And then we get our allocation!
+As a sanity check, once you are on one of the nodes you should be able to submit a job and see the output:
+
+.. code-block:: sh
+
+    noodle:~$ flux mini run hostname
+    noodle220
+    noodle221
+    noodle222
+    noodle223
+
+And you might adopt this command to be more specific to your resource manager. E.g., slurm uses srun.
```

Comment:
i would probably just remove this line, not sure if it serves much purpose at this point?  The `flux mini run hostname` is to illustrate that we're inside a flux instance.

Reply:
Remove the line about adopting the command? Or the flux mini run? I think both are important to have:

- the first recognizes that "if your hpc manager is different, you'll need to adopt this command"
- the second gives the reader a way to sanity check they are where they think they are, and a quick preview for how to run a very simple command (tying two different tutorials - the submit and ssh together).

We can see what others think. I don't think it hurts to have it, and actually might help.

Reply:
ohhh, you're referencing the `srun -N4 -n4 --pty --mpibind=off flux start` line from earlier ... i thought it was referencing the `flux mini run hostanme` example, which was why it seemed weird why you were saying it.

Perhaps we just need to move it up above the `flux mini run hostname` line.  Like

```
And then we get our allocation!  This example launches flux with slurm (i.e. salloc / srun), you may need to adjust for a different resrouce manager
```

Also, I think you meant "adapt" not "adopt"?

Reply:
yes we can move it earlier!

And good catch... I'll fix that in a minute (having some cocoa). Side note - is there a signup for adopting nodes? I can adopt them to live with my pet pusheens.